### PR TITLE
docs(readme): add PageSpeed Green to to related projects

### PR DESCRIPTION
**Pull request by alexgreencode**
**Summary**
Add [PageSpeed Green](https://pagespeed.green) to Lighthouse Integrations section.
**Related Issues/PRs**
There are no related issues/PRs

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -293,6 +293,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 * **[Lighthouse Keeper](https://lighthouse-keeper.com/)** - Lighthouse Keeper monitors your pages' Lighthouse scores and notifies you if they drop below your thresholds. Lighthouse Keeper is a free service that monitors up to 3 URLs once per day.
 
+* **[PageSpeed Green](https://pagespeed.green/)** - New web performance monitoring service for web pages. For now, you can track PageSpeed performance indicators for unlimited domains and web pages. Coming, running in different regions and CD/CI integration. Powered By Google Lighthouse and Puppeteer. You can easily start using PageSpeed Green with 30-day trial.
```
Comment (patrickhulce) [inline readme.md]:
```suggestion
* **[PageSpeed Green](https://pagespeed.green/)** - Web performance monitoring service that tracks Lighthouse performance indicators on unlimited domains and web pages.  PageSpeed Green is a paid product with a 30-day trial.
```

maybe come back to add the next sentence about different regions and CI/CD once it's implemented :)

Comment (alexgreencode) [reply readme.md]: @patrickhulce Thanks for review! Yes, it my own infrastructure. Yes, good advise about screenshots. Ok, I'll remove info about future's features.

**Comment by alexgreencode** [comment]
@patrickhulce I removed future features - please review

File: readme.md (hunk 1 above)
Comment (patrickhulce) [reply readme.md]: did you dislike the other edits? the other paid products follow the model of `X is a paid product with a Y-day trial.`

Comment (alexgreencode) [reply readme.md]: I do not like to be similar :-) Ok, I changed to the existing approach

**Pull request merged**